scripts/compactor_runner.py

```diff
@@ -282,17 +244,71 @@ def _run_corfu_compactor(self):
         except Exception as ex:
             self._print_and_log("Failed to run rsync_log " + " error: " + str(ex))
 
+        # configuring hostname and port
+        is_saas_hostname_present = False
```

Comment:
Can we move this entire block of configuring hostname and port to a separate method? There might be changes in the future and it's easier that way.

Reply:
In that case, it would also be easy to return after L253 for sass and only for on-prem we go forward

Reply:
Done, Thank you.